src/utils/trust/beneficial.owner.mapper.ts

```diff
@@ -10,6 +10,7 @@ const mapBoIndividualToPage = (
     id: beneficialOwner.id,
     name: [beneficialOwner.first_name, beneficialOwner.last_name].join(' '),
     type: BeneficialOwnerTypeChoice.individual,
+    is_newly_added: beneficialOwner.ch_reference ? false : true
```

Comment:
I've seen !! being used to convert truthy to boolean

Reply:
I had initially used !! but it was decided that this approach was safer

Reply:
After chatting with Conuir the suggestion will be used as it's the same result